src/base/utils/net.cpp

```diff
@@ -106,7 +106,7 @@ namespace Utils
             // as scope id. Furthermore, QHostAddress doesn't have any convenient method to
             // affect this, so we jump through hoops here.
             if (addr.protocol() != QAbstractSocket::IPv6Protocol)
-                return {};
+                return QHostAddress{addr.toIPv6Address()};
```

Comment:
is the explicit constructor required? would this work: `return addr.toIPv6Address();`?

Reply:
>would this work: return addr.toIPv6Address();

No. That constructor is marked as explicit.

And `return {addr.toIPv6Address()};` fails with:
```
error: converting to 'QHostAddress' from initializer list would use explicit constructor
```